api/src/opentrons/protocol_api_experimental/instrument_context.py

```diff
@@ -107,14 +113,39 @@ def pick_up_tip(
             presses: Optional[int] = None,
             increment: Optional[float] = None) -> InstrumentContext:
         # noqa: D102
-        raise NotImplementedError()
+        # TODO(al, 2021-04-12): What about presses and increment? They are not
+        #  supported by PE command. They are also not supported by PD protocols
+        #  either.
```

Comment:
I'm doing something like this for `aspirate()`'s `rate`:

```suggestion
        if (presses is not None) or (increment is not None):
            raise NotImplementedError()
```

Seems less likely to get forgotten.